subjects/atm-management-system/README.md

```diff
@@ -114,12 +114,12 @@ You can also do more bonus features or update the terminal interface:
 - Encryption of passwords
 - Adding your own Makefile
 
-A relational database can also be added as a bonus. You can use the database at your choice, but we recommend you do use SQLite
+A relational database can also be added as a bonus. You can use the database of your choice, but we recommend you use SQLite.
 To know more about SQLite you can check the [SQLite page](https://www.sqlite.org/index.html).
 
 ### Example
 
-You can find an example of the final application [here](https://www.youtube.com/watch?v=xVtikDcGG2E). Do not forget that you are free to
+You can find an example of the final application [here](https://www.youtube.com/watch?v=xVtikDcGG2E). Don't forget that you are free to
```

Comment:
I think we can accept this *for now*. Purely because there are many examples of contracted words across the content already. At some point, we can review all of those cases like "you'll", "won't" etc.